applications/crossbar/src/modules/cb_ledgers.erl

```diff
@@ -267,6 +273,48 @@ put(Context, Action) ->
 %%% Internal functions
 %%%=============================================================================
 
+%%------------------------------------------------------------------------------
+%% @doc
+%% @end
+%%------------------------------------------------------------------------------
+-spec summary(cb_context:context()) -> cb_context:context().
+summary(Context) ->
+    Options = [{'group', 'true'}
+              ,{'group_level', 0}
+              ,{'mapper', crossbar_view:map_value_fun()}
+              ,{'reduce', 'true'}
+              ,{'unchunkable', 'true'}
+              ],
+    Context1 = crossbar_view:load_modb(Context, ?VIEW_BY_TIMESTAMP, Options),
+    case cb_context:resp_status(Context1) of
+        'success' ->
+            Summary = kz_json:sum_jobjs(cb_context:doc(Context1)),
+            cb_context:set_resp_data(Context1, summary_to_dollars(Summary));
+        _ ->
+            Context1
+    end.
+
+%%------------------------------------------------------------------------------
+%% @doc
+%% @end
+%%------------------------------------------------------------------------------
+-spec account_summary(cb_context:context()) ->cb_context:context().
+account_summary(Context) ->
+    Options = [{'group', 'true'}
+              ,{'group_level', 3}
+              ,{'reduce', 'true'}
+              ,{'unchunkable', 'true'}
+              ],
+    Results = kz_datamgr:get_results(cb_context:account_modb(Context), ?VIEW_BY_ACCOUNT, Options),
```

Comment:
Use `crossbar_view` just like `summary` above(`unchunkable` is crossbar_view option btw :)